Project.toml

```diff
@@ -1,7 +1,7 @@
 name = "AutoHashEquals"
-uuid = "97026771-8bcd-44fb-b7eb-f644370d63d3"
+uuid = "15f4f7f2-30c1-5605-9d31-71845cf9641f"
 authors = ["andrew cooke <andrew@acooke.org>"]
-version = "1.0.0"
+version = "0.3.0"
```

Comment:
What's going on here?

Reply:
according to the [registry](https://github.com/JuliaRegistries/General/tree/master/A/AutoHashEquals) the last release is `0.2.0`. I think this Project.toml was autogenerated without taking into account the previous format. So I changed the uuid to match what's in the registry and bumped the version number.

Reply:
This should instead change to and release as 1.0.0

Reply:
Why 1.0.0 ? I just proposed an idea https://github.com/andrewcooke/AutoHashEquals.jl/issues/22 that I think might be good for a 1.0.0 release.

Reply:
It has been stable for a very long time and is essentially at a 1.0 level (I personally thought it had already had a 1.0 release). From a purely mechanical perspective there is no difference in impact from a 0.x to 0.y release vs a 0.x to 1.0 release, as both are "breaking".